runsc/console/pty_linux.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build linux
+// +build linux
+
+package console
+
+import "golang.org/x/sys/unix"
+
+func IsPty(fd uintptr) bool {
```

Comment:
Needs doc comment.   Also, does runsc work on OSX ?  That would surprise me.

Reply:
I originally included the `pty_darwin.go` file because `platforms_darwin.go` was explicitly included to make the platforms package (part of runsc) buildable on OSX/Darwin.

https://github.com/google/gvisor/blob/8184fa1db0faaa9cf742a6c094f40e5442635bf1/runsc/boot/platforms/platforms_darwin.go#L15-L20

I have removed the file now.